LadybugTools_Engine/Python/src/ladybugtools_toolkit/plot/_utci.py

```diff
@@ -112,9 +110,108 @@ def utci_comfort_band_comparison(
     # add labels to bars
 
     # get bar total heights
-    height = np.array([[i.get_height() for i in c] for c in ax.containers]).T.sum(
-        axis=1
-    )[0]
+    height = np.array([[i.get_height() for i in c] for c in ax.containers]).T.sum(axis=1)[0]
+    for c in ax.containers:
+        labels = []
+        for v in c:
+            label = f"{v.get_height():0.1%}" if density else f"{v.get_height():0.0f}"
+            if v.get_height() / height > 0.04:
+                labels.append(label)
+            else:
+                labels.append("")
+
+        ax.bar_label(
+            c,
+            labels=labels,
+            label_type="center",
+            color=contrasting_color(v.get_facecolor()),
+        )
+
+    ax.tick_params(axis="both", which="both", length=0)
+    ax.grid(False)
+    plt.xticks(rotation=0)
+    ax.yaxis.set_major_locator(plt.NullLocator())
+
+    return ax
+
+
+@bhom_analytics()
+def utci_comfort_band_comparison_series(
+    utci_series: tuple[pd.Series],
```

Comment:
Probably a bit of a niggle, but could this arg be called `utci_serieses`?
(or whatever the plural of series is)

Reply:
I've added a validation check for listyness and kept the variable the same name as there isn't a plural